services/meta/client.go

```diff
@@ -983,6 +983,12 @@ func snapshot(path string, data *Data) error {
 		return err
 	}
 
+	if _, err := os.Stat(file); err == nil {
```

Comment:
@jsternberg We already handle windows differences via compilation.  See https://github.com/influxdata/influxdb/blob/master/tsdb/engine/tsm1/file_windows.go